### ISSUE TITLE
no syslog on wrong username

### ISSUE THREAD

**Issue by ghost** [otacke/yalw#5]
Found a little bug. YALW/Wordpress is not logging any login attempts with a wrong username.

Here is the Problem: My workaround #2 for Buddypress checks if an user exists or not. If not you throw an error and nothing is logged by syslog.
If the user exists you try to signup and the wordpress internal logging records this by syslog.

Possible solution:
- yalw logs this attempts by himself
- remove the buddypress workaround and look for better solution
- ignore this behavior

**Comment by otacke** [comment]
Thanks!
- Number 3 is not a nice option IMHO :-) 
- Number 2 is totally viable but, I have no coding experience with Buddypress yet, so this might take a while. You're welcome to file another pull request.
- Number 1 might not look elegant, but it'd be a quick win. Of course, you're welcome to file a pull request as well ;-)

What do you think? Nice working with you.

**Comment by ghost** [comment]
The whole problem exists only when someone uses wp-fail2ban with yalw. The moment someone tries to login as a regular user he get tracked by wp-fail2ban. If someone tries login as a non existing user he wont get tracked by yalw or wp-fail2ban. 

On a system with yalw and wp-fail2ban someone can brute free user names without consequences until he hits a user name that is taken. So i would say: Number 1 - ignore this bug :D

**Comment by otacke** [comment]
From a security driven point of view you're right. Still, it might be interesting performance-wise.

If you really want to brute-force a system, you'd probably launch requests in parallel and this could influence the server performance. Not sure about the magnitude though. One of the nice aspects of fail2ban is that it stops attacks on a low level before the webserver kicks in. On the other hand, if someone wants to DDoS a server, he or she wouldn't have to use the login procedure...

Then again, knowledge about existing user names could possibly be interesting for social engineering, and even gaining a valid user name could be a valid goal. But that might be "too academic" and you're probably right that this issue can be ignored quite safely.
